Composer/packages/server/src/models/settings/interface.ts

```diff
@@ -4,7 +4,7 @@
 export const OBFUSCATED_VALUE = '*****';
 
 export interface ISettingManager {
-  get(slot?: string, obfuscate?: boolean): Promise<any | null>;
-  set(slot: string, settings: any): Promise<void>;
+  get(obfuscate?: boolean): Promise<any | null>;
```

Comment:
Should we also take the chance of remove this `obfuscate` feature, @cwhitten what do you think? I think we chatted about this

Reply:
Let's leave it in for now ad revisit once some customers have successfully deployed Composer and we get some additional feedback.